src/codegen/compile.ml

```diff
@@ -1534,11 +1627,24 @@ module BitTagged = struct
      tell it apart from a pointer by looking at the last bits: if set, it is a
      pointer.
 
-     Small here means -2^30 ≤ x < 2^30, and untagging needs to happen with an
-     _arithmetic_ right shift. This is the right thing to do for signed
-     numbers, and because we want to apply a consistent logic for all types,
-     especially as there is only one wasm type, we use the same range for
-     signed numbers as well.
```

Comment:
Was that a typo?

"we use the same range for  signed numbers as well."|-> 
"we use the same range for *un*signed numbers as well."

That always confused me....